src/plugins/discover/public/application/components/table/table.tsx

```diff
@@ -142,10 +146,12 @@ export function DocViewTable({
                           displayUnderscoreWarning={displayUnderscoreWarning}
                           isCollapsed={isCollapsed}
                           isCollapsible={isCollapsible}
-                          isColumnActive={Array.isArray(columns) && columns.includes(field)}
+                          isColumnActive={Array.isArray(columns) && columns.includes(multiField)}
```

Comment:
why do we need this check if columns is an array? it will always be at least an empty array, won't it?

Reply:
no, it can be `undefined` in the doc view, however, no need to check for an array here, since it's an array or undefined, good catch!